src/components/NavigationTree/NavigationTree.tsx

```diff
@@ -13,7 +13,7 @@ export function NavigationTree({
     getActions,
     activePath,
     onActivePathUpdate,
-    cache,
+    cache = true,
```

Comment:
this displays the default value in storybook, whereas the previous version does not